lib/bemhtml/index.js

```diff
@@ -130,10 +130,10 @@ BEMHTML.prototype.render = function render(context,
     }
 
     if (cls)
-      out += ' ' + cls;
+      out += ' ' + (cls.trim ? utils.attrEscape(cls).trim() : cls);
```

Comment:
if you use assert for `cls.trim` just to check if it's an instance of String, why not to do it in more intuitive way: `typeof cls === 'string'`?